proseco/tests/test_catalog.py

```diff
@@ -280,6 +280,88 @@ def test_big_sim_offset():
     assert all(name in aca.fids.colnames for name in names)
 
 
+@pytest.mark.parametrize('call_t_ccd', [True, False])
+def test_calling_with_t_ccd_acq_guide(call_t_ccd):
+    """Test that calling get_aca_catalog with t_ccd or t_ccd_acq/guide args sets all
+    CCD attributes correctly in the nominal case of a temperature
+    below the planning limit.
+
+    """
+    dark = DARK40.copy()
+    stars = StarsTable.empty()
+    stars.add_fake_constellation(mag=8.0, n_stars=8)
+
+    t_ccd = np.trunc(ACA.aca_t_ccd_planning_limit - 1.0)
+
+    if call_t_ccd:
+        # Call with just t_ccd=t_ccd
+        t_ccd_guide = t_ccd
+        t_ccd_acq = t_ccd
+        ccd_kwargs = {'t_ccd': t_ccd}
+    else:
+        # Call with separate values
+        t_ccd_guide = t_ccd
+        t_ccd_acq = t_ccd - 1
+        ccd_kwargs = {'t_ccd_acq': t_ccd_acq, 't_ccd_guide': t_ccd_guide}
+
+    kwargs = mod_std_info(stars=stars, dark=dark, **ccd_kwargs)
```

Comment:
@jeanconn - look here and at the tests below  Hopefully at least the API now makes sense.  The implementation is not that tricky either but there are some obfuscations that must be traversed...